src/paths/components/create.yaml

```diff
@@ -2,6 +2,8 @@ summary: Create a new component
 tags:
   - components
 operationId: componentsCreate
+security:
+  - bearerAuth: ["COMPONENTS_WRITE"]
 requestBody:
   description: The new Component to be created
   required: true
```

Comment:
All secured endpoints should have a 401 and a 403 response: 401 when a no/bad/expired token is provided, 403 when the authenticated user does not have the right privileges (https://stackoverflow.com/questions/3297048/403-forbidden-vs-401-unauthorized-http-responses#:~:text=In%20summary%2C%20a%20401%20Unauthorized,operation%20on%20the%20given%20resource.)

Reply:
It's strange, the server returns me 401 error where the token is expired or bad  but 403 when there is no token.

Reply:
issue created